variants/aws-dev/Cargo.toml

```
@@ -12,7 +12,8 @@ kernel-parameters = [
     "console=tty0",
     "console=ttyS0,115200n8",
     # Only reserve if there are at least 2GB
-    "crashkernel=2G-:256M"
+    "crashkernel=2G-:256M",
+    "net.ifnames=0"
```

nit: this would be a good time to add the trailing comma to all list elements, since that's more friendly for future diffs
```suggestion
    "crashkernel=2G-:256M",
    "net.ifnames=0",
```